clients/go/cmd/zbctl/main_test.go

```diff
@@ -214,12 +214,12 @@ func (s *integrationTestSuite) TestCommonCommands() {
 			if test.jsonOutput {
 				cmdOut, err = reformatJSON(cmdOut)
 				if err != nil {
-					t.Fatalf("failed to reformat JSON: %v", err)
+					t.Fatalf("failed to reformat response JSON: %v\nErroneous JSON: %v", err, cmdOut)
```

Comment:
@npepinpe I think this doesn't work as intended, since we overwrite in the previous statement the cmdOut

See an example here https://go.dev/play/p/aLcYSiUjWvp this caused to output always `[]` https://github.com/camunda-cloud/zeebe/issues/8284#issuecomment-1004873768